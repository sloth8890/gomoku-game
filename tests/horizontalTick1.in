place A1
place B10
place B1
place C2
place C1
place C4
place D1
place D19
place E1
