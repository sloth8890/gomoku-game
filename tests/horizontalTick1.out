Black wins!
A1B10B1C2C1C4D1D19E1
Thank you for playing!
